### ISSUE TITLE
new window doesn't inhreit the colors

### ISSUE THREAD

**Issue by ixzh** [thevinter/styli.sh#22]
It seems to onlychange the existing window with -p option.
How to make new terminal instances inherit the colors?

**Comment by thevinter** [comment]
What terminal are you using?

**Comment by ixzh** [comment]
> What terminal are you using?

Alacritty

**Comment by thevinter** [comment]
Alacritty has its own way of managing the colors, I merged some changes and now it should work natively, let me know if you have other problems.

**Comment by ixzh** [comment]
> I merged some changes and now it should work natively

So Alacritty can't inherit the colors? I pulled the latest , still new instance has the default colors.

![image](https://user-images.githubusercontent.com/14981695/122677555-68d70400-d215-11eb-9b55-1cef0d63834b.png)

**Comment by thevinter** [comment]
Alacritty isn't supported natively by pywal so I'm directly modifying the `alacritty.yml` in my code

Can you please check if after running `styli.sh -p` the colour values in your config file change?

**Comment by thevinter** [comment]
Ok, it was a stupid bug. It should properly work now. Let me know :)

**Comment by ixzh** [comment]
> Can you please check if after running `styli.sh -p` the colour values in your config file change

you mean my alacrity.yaml in the .config folder? I'll check but if it changes the new window should follow new color scheme? my above screenshot, top left window is a new alacritty window with old color, whereas the existing ones are updated to new colors. should new opened windows with the new generated colors as well🤔🤔

**Comment by thevinter** [comment]
Yes, that's exactly what the last update pushed. Try updating to the new version and run it again. If it doesn't work try commenting the already existing colors in the `alacritty.yml`. It works on my machine with alacritty installed.

**Comment by ixzh** [comment]
this is the ~/.config/alacritty.yml, have I commented out the right section?  As it doesn't affect new terminal color, even after `styli.sh -p`:
![image](https://user-images.githubusercontent.com/14981695/122699822-b76bb900-d27c-11eb-81e9-fba8b29a5c92.png)

**Comment by thevinter** [comment]
Could you please upload your entire `alacritty.yml` to a pastebin? Thanks

**Comment by ixzh** [comment]
> Could you please upload your entire `alacritty.yml` to a pastebin? Thanks

Hi, attached herein. It is the sample yml file provided in the repo, I did a few changes only.
https://pastebin.com/CNaEVWjc

**Comment by ixzh** [comment]
I notice the following line:
```
DEFAULT_MACOS_CONFIG="$HOME"/.config/alacritty/alacritty.yml
```
my file was .config/alacritty.yml, one folder less.  So I guess this will make DEFAULT_MACOS_CONFIG not exist, but still existing terminals can be updated to new colors. So the terminal color scheme is not determined by this .config/alacritty.yml ?

**Comment by thevinter** [comment]
Yes, that's the problem. If you change that line to point to your config file it should work. 

Sadly alacritty and pywal [have been known not to work well together](https://github.com/alacritty/alacritty/issues/4220) so I had to implement a workaround. Let me know if changing the `DEFAULT_MACOS_CONFIG` path works for you.

**Comment by ixzh** [comment]
still no luck... maybe wait and see how it goes 😜😜😜 keep on exploring 🙂
